compiler/rustc_apfloat/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 authors = ["The Rust Project Developers"]
 name = "rustc_apfloat"
 version = "0.0.0"
+license = "MIT OR Apache-2.0"
```

Comment:
Can this be `MIT OR Apache-2.0`? It was fairly straightforwardly ported from the LLVM apfloat library. Maybe Apache 2.0 with llvm exception instead? SPDX License Identifier: "Apache-2.0 WITH LLVM-exception" according to https://llvm.org/foundation/relicensing/ .